.github/workflows/push-dist.yml

```
@@ -0,0 +1,20 @@
+name: Push dist
+
+on:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  push-dist:
+    name: Push dist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/pnpm
+      - uses: kategengler/put-built-npm-package-contents-on-branch@v1.0.0
+        with:
+          branch: dist
+          token: ${{ secrets.PAT }}
```

The GITHUB_TOKEN on this repo has write access, if you want to swap this out
```suggestion
          token: ${{ secrets.GITHUB_TOKEN }}
```